src/_scss/pages/homepage/features/_featurePane.scss

```diff
@@ -44,33 +44,42 @@
         @include flex-wrap(wrap);
     }
 
-    .feature-pane__content {
-      @include display(flex);
-      @include flex-direction(column);
-      text-align: center;
-      font-size: rem(14);
-      line-height: rem(17);
-      @include media($medium-screen) {
-          @include flex(1 1 1);
-          width: 30%;
-      }
-      &.feature-pane__content-middle__sub {
-        background-size: 100% 100%;
-        background-repeat: no-repeat;
-        background-image: url('#{$image-path}/homepage-eagle-large.png');
-      }
+
+  .feature-pane__content {
+    @include display(flex);
+    @include flex-direction(column);
+    text-align: center;
+    font-size: rem(14);
+    line-height: rem(17);
+    @include media($medium-screen) {
+      @include flex(1 1 1);
+      width: 30%;
     }
-    
-    .feature-pane__content-divider {
-        display: none;
-        @include media($medium-screen) {
-            display: block;
-            border-right: 1px solid $color-gray-lighter;
-            height: 202px;
-            align-self: center;
-        }
-        
+
+    &.feature-pane__content-middle__sub {
+      background-size: cover;
+      background-position: center center;
+      background-repeat: no-repeat;
+      background-image: url('#{$image-path}/homepage-eagle-large.png');
     }
+    &.feature-pane__content-fiscal-data {
```

Comment:
Class name `feature-pane__content_fiscal-data` would be more consistent with our BEM convention (`fiscal-data` is the modifier)